pavelib/assets.py

```diff
@@ -378,7 +378,7 @@ def collect_assets(systems, settings):
     `settings` is the Django settings module to use.
     """
     for sys in systems:
-        sh(django_cmd(sys, settings, "collectstatic --noinput > /dev/null"))
+        sh(django_cmd(sys, settings, "collectstatic --clear --noinput -v 0"))
```

Comment:
@andy-armstrong  Oh, -v0 is very very close to >/dev/null but lets out a few warnings in the code that were being hidden but should get fixed.  I can get you a gist of running it and you can comment on it.